CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+
+Every significant change is documented in the changelog file.
+
+# Branch Organization
+
+Submit all changes directly to the master branch. We don’t use separate branches for development or for upcoming releases.
+
+Code merged to master must be compatible with the latest stable release. It may contain additional features, but no breaking changes. We should be able to release a new minor version from the tip of master at any time.
+
+# Features, Enhancements & Bugs
+
+We use GitHub Issues for all of our feature requests, enhancements, and bugs.
+
+## Where to Find Known Issues
+
+If you are experiencing an issue you can check our GitHub Issues. All issues related to known bugs will be labeled as 'bug'. We keep a close eye on this and try to make it clear when we have an internal fix in progress.
+
+## Reporting New Issues
+
+Before filing a new task, try to make sure your problem does not already exist by looking through the known issues. If you are experiencing a problem that you think is not documented, create an issue and attach the 'Bug' label.
+
+Before fixing a bug we need to reproduce and confirm it. In order to reproduce bugs, we require that you provide a reproducible scenario. Having a minimal reproducible scenario gives us important information without going back and forth to you with additional questions.
+
+## Security Bugs
+
+If you have encountered a security vulnerability, then create an issue and attach the 'Security' label.
+
+# Proposing a Change
+
+If you intend to make any non-trivial changes to the implementation, we recommend filing an issue. This lets us reach an agreement on your proposal before you put significant effort into it.
+
+If you’re only fixing a bug, it’s fine to submit a pull request right away, but we still recommend that you file an issue detailing what you’re fixing. This is helpful in case we don’t accept that specific fix but want to keep track of the issue.
+
+# Documentation
+
+When a change is made please update the documentation found in the `README.md` files accordingly so that the documentation reflects the code.
+
+# Submitting a pull request
+
+The core team is monitoring for pull requests. We will review your pull request and either merge it, request changes to it, or close it with an explanation. We’ll do our best to provide updates and feedback throughout the process.
+
+## Before Submitting
+
+Please make sure the following is done before submitting a pull request:
+
+1. Fork the repository and create your branch from master.
+2. If you’ve fixed a bug or added code that should be tested, then add tests.
+3. Ensure the test suite passes by running `make test`.
+4. Format your code with `make format`.
+5. Lint your code with `make lint`.
+6. If you haven’t already, complete the CLA.
+7. Make sure your pull request describes the issue you are fixing, or the feature you are adding.
```

Comment:
I feel like we should we provide better guidance around the issue/PR relationship. For example, "Fixes #"

Reply:
That's a good point. I'm working on adding a section for Axway contributors. For an OS contribution, not sure if they should add that comment, or if we should do it as the maintainers.